grail_buckets/README.md

```diff
@@ -0,0 +1,19 @@
+# Grail Bucket definitions
+
+This is a simple example on how to manage grail bucket definitions.
```

Comment:
nit: might be nice to link to the docs here: https://www.dynatrace.com/support/help/platform/grail/data-model#custom-grail-buckets